assets/src/js/helper.js

```diff
@@ -98,6 +105,20 @@ function dokan_get_i18n_date_format( format = true ) {
   return timeFormat;
 }
 
+/**
+ * Check format should convert or not.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @param {object}         formats Time format object
+ * @param {string|boolean} format  Format
+ *
+ * @returns {string} Return format
+ */
+function get_formatted_string( formats, format ) {
```

Comment:
we don't need a global function for this, kindly move this function under  dokan_get_formatted_time()